test/bigtest/tests/mapping-profiles/new-mapping-profiles-test.js

```diff
@@ -149,6 +149,10 @@ describe('Mapping profile form', () => {
               expect(mappingProfileForm.marcDetailsTable.rows(0).cells(8).hasContent()).to.be.false;
             });
 
+            it('trash button is disabled as there is only one row present', () => {
```

Comment:
I would suggest relying on the feature in the description rather than on the look which is now trash but later can be a cross and thus would make the message irrelevant.
```suggestion
            it('remove row button is disabled as there is only one row present', () => {
```